docs/microsoft-365-developer-program-faq.yml

```diff
@@ -188,12 +190,12 @@ sections:
       - question: |
           Can I share my developer subscription with my team?
         answer: |
-          As a member of the Microsoft 365 Developer Program, you are entitled to one free Microsoft 365 developer subscription. Your subscription is associated with your developer program account. You can't transfer owership of your subscription to a different account.  
+          Qualified members of the Microsoft 365 Developer Program are entitled to one Microsoft 365 E5 developer subscription. If you qualify for a subscription, your subscription is associated with your developer program account. You can't transfer owership of your subscription to a different account.  
```

Comment:
Qualified members of the Microsoft 365 Developer Program are entitled to one Microsoft 365 E5 developer subscription. If you qualify for a subscription, your subscription **will be** associated with your developer program account. You **may not transfer ownership** to a different account.